onnx/defs/tensor/defs.cc

```diff
@@ -1938,6 +1940,17 @@ ONNX_OPERATOR_SET_SCHEMA(
               fail_type_inference(
                   "Input 'depth' must have exactly one element.");
             }
+            const TensorProto* depth_initializer = ctx.getInputData(1);
+            if (depth_initializer != nullptr) {
+              if (depth_initializer->has_raw_data()) {
```

Comment:
Should this also check int64_data and int32_data if the value isn't specified in raw_data?